### PR TITLE
Main page + About page

### PR DESCRIPTION
**Pull request by superpuper777**
This PR for only team-review to identify gaps in the development process (only for full-screen).

### CODE REVIEW

File: rslang/src/app/main/team/team.html [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+            <img
+              src="https://us.123rf.com/450wm/alexutemov/alexutemov1605/alexutemov160500383/56523661-la-felicidad-concepto-de-la-gente-la-cara-joven-hombre-de-la-barba-feliz-vector-de-la-cara-y-la-cara.jpg?ver=6"
+              alt=""
+            />
+          </div>
+          <mat-card-title class="main__our_team__people__name"
+            >Name Surname</mat-card-title
+          >
+          <mat-card-subtitle class="main__our_team__people__position"
+            >Team Lead</mat-card-subtitle
```
Comment (Nikitzu) [inline rslang/src/app/main/team/team.html]: Not good code style. return closed bracket on the previous line

File: rslang/src/app/main/team/team.html [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+                alt="github_icon"
+              />
+            </a>
+          </div>
+        </mat-card>
+
+        <mat-card class="main__our_team__people__card medium">
+          <div class="main__our_team__people__photo">
+            <img
+              src="https://us.123rf.com/450wm/alexutemov/alexutemov1605/alexutemov160500383/56523661-la-felicidad-concepto-de-la-gente-la-cara-joven-hombre-de-la-barba-feliz-vector-de-la-cara-y-la-cara.jpg?ver=6"
```
Comment (Nikitzu) [inline rslang/src/app/main/team/team.html]: we should not have such long links

File: rslang/src/app/main/team/team.html [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+                alt="github_icon"
+              />
+            </a>
+          </div>
+        </mat-card>
+
+        <mat-card class="main__our_team__people__card">
+          <div class="main__our_team__people__photo">
+            <img
+              src="https://us.123rf.com/450wm/alexutemov/alexutemov1605/alexutemov160500383/56523661-la-felicidad-concepto-de-la-gente-la-cara-joven-hombre-de-la-barba-feliz-vector-de-la-cara-y-la-cara.jpg?ver=6"
```
Comment (Nikitzu) [inline rslang/src/app/main/team/team.html]: same

File: rslang/src/app/main/team/team.html [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+            </a>
+          </div>
+        </mat-card>
+      </div>
+
+      <div class="group_2">
+        <mat-card class="main__our_team__people__card first">
+          <div class="main__our_team__people__photo">
+            <img
+              src="https://us.123rf.com/450wm/alexutemov/alexutemov1605/alexutemov160500383/56523661-la-felicidad-concepto-de-la-gente-la-cara-joven-hombre-de-la-barba-feliz-vector-de-la-cara-y-la-cara.jpg?ver=6"
```
Comment (Nikitzu) [inline rslang/src/app/main/team/team.html]: same

File: rslang/src/app/main/team/team.html [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+                alt="github_icon"
+              />
+            </a>
+          </div>
+        </mat-card>
+
+        <mat-card class="main__our_team__people__card second">
+          <div class="main__our_team__people__photo">
+            <img
+              src="https://us.123rf.com/450wm/alexutemov/alexutemov1605/alexutemov160500383/56523661-la-felicidad-concepto-de-la-gente-la-cara-joven-hombre-de-la-barba-feliz-vector-de-la-cara-y-la-cara.jpg?ver=6"
```
Comment (Nikitzu) [inline rslang/src/app/main/team/team.html]: same

File: rslang/src/app/main/team/team.html [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+              />
+            </a>
+          </div>
+        </mat-card>
+
+        <div class="group_3">
+          <mat-card class="main__our_team__people__card third">
+            <div class="main__our_team__people__photo">
+              <img
+                src="https://us.123rf.com/450wm/alexutemov/alexutemov1605/alexutemov160500383/56523661-la-felicidad-concepto-de-la-gente-la-cara-joven-hombre-de-la-barba-feliz-vector-de-la-cara-y-la-cara.jpg?ver=6"
```
Comment (Nikitzu) [inline rslang/src/app/main/team/team.html]: same

File: rslang/src/app/main/team/team.html [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+                  src="../../../../assets/images/github_icon.png"
+                  alt="github_icon"
+                />
+              </a>
+            </div>
+          </mat-card>
+          <mat-card class="main__our_team__people__card fourth">
+            <div class="main__our_team__people__photo">
+              <img
+                src="https://us.123rf.com/450wm/alexutemov/alexutemov1605/alexutemov160500383/56523661-la-felicidad-concepto-de-la-gente-la-cara-joven-hombre-de-la-barba-feliz-vector-de-la-cara-y-la-cara.jpg?ver=6"
```
Comment (Nikitzu) [inline rslang/src/app/main/team/team.html]: same

File: rslang/src/app/main/team/team.scss [hunk 8]
```diff
@@ -0,0 +1,104 @@
+@import "src/variables";
+
+.main__our_team {
```
Comment (Nikitzu) [inline rslang/src/app/main/team/team.scss]: why __ and then _?

File: rslang/src/app/material/material.module.ts [hunk 9]
```diff
@@ -1,19 +1,33 @@
+import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { MatMenuModule } from '@angular/material/menu';
 import { MatToolbarModule } from '@angular/material/toolbar';
 import { MatIconModule } from '@angular/material/icon';
 import { MatButtonModule } from '@angular/material/button';
 import { MatDialogModule } from '@angular/material/dialog';
 import { MatCheckboxModule } from '@angular/material/checkbox';
-
+import { MatCardModule } from '@angular/material/card';
```
Comment (Nikitzu) [inline rslang/src/app/material/material.module.ts:9]: remove. module will be deprecated

File: rslang/src/app/material/material.module.ts [hunk 10]
```diff
@@ -1,19 +1,33 @@
...
+  imports: [
+    CommonModule,
+    MatMenuModule,
+    MatToolbarModule,
+    MatIconModule,
+    MatButtonModule,
+    MatDialogModule,
+    MatCheckboxModule,
+    MatCardModule,
+  ],
```
Comment (Nikitzu) [inline rslang/src/app/material/material.module.ts:21]: same

**Review by Nikitzu — CHANGES_REQUESTED** [review]

File: rslang/src/app/main/team/team.html [hunk 11]
```diff
@@ -0,0 +1,20 @@
+<section class="main__our__team">
+  <h3 class="main__our__team__title">OUR TEAM</h3>
```
Comment (coriander31415) [inline rslang/src/app/main/team/team.html:2]: Center the title, please

File: rslang/src/app/main/components/main-page/main-page.component.html [hunk 12]
```diff
@@ -1 +1,80 @@
...
+      >
+      </iframe>
+    </div>
+  </div>
+</section>
+
+<app-team></app-team>
+
+<section class="main-page__advantages">
+  <h3 class="main-page__advantages__title">OUR ADVANTAGES</h3>
```
Comment (coriander31415) [inline rslang/src/app/main/components/main-page/main-page.component.html:33]: Center the title, please

**Review by coriander31415 — CHANGES_REQUESTED** [review]

**Comment by whattablackhole** [comment]
not quite responsive
Screenshot from 2021-04-03 21-57-22

**Comment by superpuper777** [comment]
> not quite responsive
> Screenshot from 2021-04-03 21-57-22

**Pull request closed without merging**